DigitalOffice/Tests/ProjectServiceTests/Commands/EditProjectByIdCommandTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿using Moq;
+using System;
+using NUnit.Framework;
+using FluentValidation;
+using LT.DigitalOffice.ProjectService.Models;
+using LT.DigitalOffice.ProjectService.Commands;
+using LT.DigitalOffice.ProjectService.Database.Entities;
+using LT.DigitalOffice.ProjectService.Mappers.Interfaces;
+using LT.DigitalOffice.ProjectService.Commands.Interfaces;
+using LT.DigitalOffice.ProjectService.Repositories.Interfaces;
+
+namespace LT.DigitalOffice.ProjectServiceUnitTests.Commands
+{
+    public class EditProjectByIdCommandTests
+    {
+        private Guid projectId;
+
+        private DbProject mappedRequest;
+
+        private EditProjectRequest editRequest;
+
+        private IEditProjectByIdCommand command;
+
+        private Mock<IProjectRepository> repositoryMock;
+        private Mock<IValidator<EditProjectRequest>> validatorMock;
+        private Mock<IMapper<EditProjectRequest, DbProject>> mapperMock;
+
+        [SetUp]
+        public void SetUp()
+        {
+            projectId = Guid.NewGuid();
+
+            validatorMock = new Mock<IValidator<EditProjectRequest>>();
+            repositoryMock = new Mock<IProjectRepository>();
+            mapperMock = new Mock<IMapper<EditProjectRequest, DbProject>>();
+
+            command = new EditProjectByIdCommand(mapperMock.Object, validatorMock.Object, repositoryMock.Object);
+
+            editRequest = new EditProjectRequest
+            {
+                Name = "DigitalOffice",
+                DepartmentId = Guid.NewGuid(),
+                Description = "A new description",
+                IsActive = false
+            };
+
+            mappedRequest = new DbProject
+            {
+                Name = editRequest.Name,
+                DepartmentId = editRequest.DepartmentId,
+                Description = editRequest.Description,
+                IsActive = editRequest.IsActive
+            };
+        }
+
+        [Test]
+        public void FailedEditProjectIncorrectDataTest()
+        {
+            validatorMock
+                    .Setup(x => x.Validate(It.IsAny<IValidationContext>()).IsValid)
+                    .Returns(false);
+
+            Assert.Throws<ValidationException>(
```

Comment:
Add verifying for mocks

Reply:
Fixed

Reply:
Not fixed :(

Reply:
Weird. I'll re-check

Reply:
Wait. That's not weird.
Verifying for mocks are added where needed.

Ah, I see... You mean to add verification that they were never called. Alright, that's my bad.